object-manipulation-2/src/pick.js

```diff
@@ -0,0 +1,12 @@
+/* exported pick */
+function pick(source, keys) {
+  var object = {};
+  for (var i = 0; i < keys.length; i++) {
+    for (var key in source) {
+      if (key === keys[i] && source[key] !== undefined) {
```

Comment:
Rather than doing `source[key] !== undefined` with each iteration, you should check for that first. Only run the `for...in` loop _if_ `source[key] !== undefined`